vscode/src/jsonrpc/agent-protocol.ts

```diff
@@ -198,6 +198,10 @@ export type ClientRequests = {
     // Returns diagnostics for the given URI. Lives under `testing/` instead of
     // standalone `diagnostics/` because it only works for TypeScript files.
     'testing/diagnostics': [{ uri: string }, { diagnostics: ProtocolDiagnostic[] }]
+    'autocomplete/tscContextRetriever': [
```

Comment:
Just to be clear this is intended for testing purposes
```suggestion
    'testing/tsc/retrieveContext': [
```

Reply:
done